src/NuGet.Clients/VsExtension/NuGetPackage.cs

```diff
@@ -196,12 +196,30 @@ private ISolutionManager SolutionManager
                 if (_solutionManager == null)
                 {
                     _solutionManager = ServiceLocator.GetInstance<ISolutionManager>();
+                    _solutionManager.AfterNuGetProjectRenamed += SolutionManager_NuGetProjectRenamed;
```

Comment:
Can this be added at the same spot in PackageManagerControl as where the other events are hooked up?